pom.xml

```diff
@@ -59,6 +76,18 @@
       <artifactId>marc4j</artifactId>
       <version>2.9.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.19.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>2.10.9</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
As I understand it, joda time is no longer intended to be used, and instead the Java 8+ date time classes used.

Why has this dependency been introduced?

Reply:
It was necessary for this pr branch initially because it was used by RMB and this module used it. Perhaps not anymore as this branch is old by now

Reply:
mod-inventory-storage has been using joda-time in tests since 2019:
https://github.com/folio-org/mod-inventory-storage/commit/2d0737522f0c1681411b0143d17558b3c13ff30e
https://github.com/folio-org/mod-inventory-storage/search?q=joda
Jodat-time usage is a technical depth of mod-inventory-storage: MODINVSTOR-733
Removing joda-time usage from mod-inventory-storage is out of scope of this pull request.
The explicit joda-time dependency in pom.xml is needed because RMB domain-models-runtime before 33.0.0 shipped with joda-time. Since 33.0.0 joda-time is no longer included.

Reply:
@julianladisch Thank you for explaining why these dependencies were introduced. At no point did I suggest including the removal of joda time in the scope of this pull request.

(this comment was made redundant by the other pull request)